packages/components/src/FormatFile/FormatFile.test.tsx

```diff
@@ -33,6 +33,26 @@ it("renders an image when provided as src", async () => {
   expect(await findByRole("img")).toBeInTheDocument();
 });
 
+it("renders a skeleton loader when provided an image", async () => {
```

Comment:
I think the wording could be updated for this test.

it("renders a skeleton loader when the provided image has not fully loaded") - something to indicate the lag